addons/website_slides/models/slide_channel.py

```diff
@@ -86,6 +86,7 @@ def _default_access_token(self):
     name = fields.Char('Name', translate=True, required=True)
     active = fields.Boolean(default=True)
     description = fields.Text('Short Description', translate=True)
+    description_card = fields.Text(translate=True) # The description that is displayed on the course card.
```

Comment:
Name it description_short (to be more explicit).

In fact I would probably put the whole logic into a `compute` method, as description_short depends directly from description. So you won't need to add the logic in write AND create. And it would be clearer about what is exactly this field and what it does. A `stored` one, obviously
And I agree with @tde-banana-odoo , description_short should be the x first char from description (with an ellipsis maybe in the template) by default, and the field can be modified.

Use a char field instead with size=x ,so it will always have the same maximum size (from compute or write)

We always prefer to give a display name to a field, even if it's the same than the field name. (if you follow me :D)
Also, the comment is good but put it into the help of the field, so not only the dev can see it, it could be useful for users to.

At the end, this should looks like :
```
description_short = fields.Char("Short Description", size=xxx,  compute="_compute_description_short", store=True,
                                translate=True, help="The description that is displayed on the course card")
```

Reply:
As discussed yesterday, it's not actually possible to do it in a `compute` method. The `write` of the `description` field is executed before the `compute`, and it's not possible to access the previous version of `description` when the `compute` is executed.

However, I did rename `description_card` to `description_short` and added the comment inside the `help` attribute.
I also added this field to the backend view.